### DISCUSSION TITLE
Renovate should support *.MODULE.bazel pattern

### DISCUSSION THREAD

**Discussion by daniel-b2c2** [renovatebot/renovate#31916] [Suggest an Idea]
### Tell us more.

[In the context of Bazel support]

Renovate supports `MODULE.bazel` pattern but treats it as an exact file, rather than a suffix, and so does not support `*.MODULE.bazel`

While it's common for most projects to start with a single bazel module file `MODULE.bazel`, larger monorepos often refactor and segregate this file into sub-modules, `blah.MODULE.bazel`.

Renovate debug logs show a file search for the pattern (^|/)MODULE\.bazel$ which is too strict but could be modified to support `blah.MODULE.bazel`

Minimum demonstration project showing how renovate currently doesn't support this: https://github.com/daniel-b2c2/bazel-module-issue

Proof that I'm not just making this up `MODULE.bazel` is a suffix and so `*.MODULE.bazel` is permitted: https://github.com/bazelbuild/bazel/pull/22075/files

We've been using renovate for a while, but since we've had to refactor the MODULE.bazel file, we're no longer receiving pull requests as the subfile are not detected as bazel modules. 

Many thanks for considering this extension.

**Comment by rarkins** [comment]
Converted to issue: https://github.com/renovatebot/renovate/issues/31918
